### DISCUSSION TITLE
Min setting for relational fields

### DISCUSSION THREAD

**Discussion by terryupton** [craftcms/cms#8918] [Ideas]
### Description
Not sure if there is a feature request already for this and it seems like a potential common issue.

But it would be great to be able to set the minimum amount of entries required for a content field type.
Similar to how matrix blocks work with a min and max blocks. The same would apply to entries instead of the current approach of a single count that restricts the maximum.  

There are several use cases I have come across for this, whereby you might need the content editor or choose a set number, this would also hep considerably with templating and simplification in some instances.

**Comment by brandonkelly** [comment]
I’m not sure I’m following… you want to set a min/max number of entries that can be added to a channel?

**Comment by terryupton** [comment]
Hey @brandonkelly - sorry it wasn't clear.
No I am talking about adding content to an entries (content) field type. 
https://craftcms.com/docs/3.x/entries-fields.html

So you can specify the min max amount of entries that can be selected, rather than just a single limit field, which is essentially a Max option.

**Comment by thomas-dm** [comment]
This would be a nice addition indeed!
We’ve also come across a few situations where requiring a minimum amount of entries for some Entries fields would’ve been ideal.
